src/components/RadioButton/RadioButton.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+  <div class="ph-radio-item" :class="[disabled && 'ph-radio-disabled']">
+    <input
+      :id="id"
+      ref="radioInputItem"
+      type="radio"
+      class="ph-radio-input"
+      :name="name"
+      :checked="checked"
+      @click="$emit('click')"
+    />
+    <label
+      :for="id"
+      class="ph-radio-label"
+      :class="icon && 'ph-radio-bar-bottom-brand'"
+    >
+      <div :class="icon && 'ph-radio-label-icon'">
+        <p-icon v-if="icon" :name="icon" type="xl" class="ph-radio-svg-icon" />
+        <span class="ph-radio-label-text">{{ label }}</span>
+      </div>
+    </label>
+
+    <div v-if="!icon" class="ph-radio-circle-container">
+      <div class="ph-radio-circle" />
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue, { PropType } from 'vue';
+import PIcon from '@/components/Icon';
+
+export default Vue.extend({
+  name: 'InputRadio',
+  components: {
+    PIcon,
+  },
+  props: {
+    name: {
+      type: String as PropType<string>,
+      default: '',
+    },
+    label: {
+      type: String as PropType<string>,
+      default: '',
+    },
+    id: {
+      type: String as PropType<string>,
+      default: '',
+    },
+    icon: {
+      type: String as PropType<string>,
+      default: '',
+    },
+    disabled: {
+      type: Boolean as PropType<boolean>,
+      default: false,
+    },
+    checked: {
+      type: Boolean as PropType<boolean>,
+      default: false,
+    },
+  },
+});
+</script>
+
+<style lang="scss" scoped>
+.ph-radio-item {
+  position: relative;
+}
```

Comment:
**issue:** use nested scss classes.

In order to be consistend we should use nested scss classes. 

Try to put all the classes in a nested tree like this:

```scss
.ph-radio-item {
  position: relative;
  
  .ph-radio-label {
    // ... your regular code ...
  }
  
  &.ph-radio-disabled {
    .ph-radio-label {
      // ... your disabled state code ...
    }
  }

}
```